docs/cosmos-based/parser/migrations/v3.0.0.md

```diff
@@ -109,8 +109,23 @@ database:
 ```
 Read [Database Config Reference](./../config/config.md#database) for more information.
 
+2. Add custom address parser inside `bdjuno/database/migrate/utils.go` file, 
+or leave it empty if no custom address parser is needed (optional).
+```go
+// If no custom address parser
+var CustomAccountParser = []string{}
+```
+```go
+// Take desmos chain as example:
+var CustomAccountParser = []string{
+  "sender", "receiver", "user", "counterparty", "blocker", "blocked",
+}
+```
+
```

Comment:
We have an issue here because migration codes are moved into juno.. 

use will have to fork `juno` and make changes [here](https://github.com/forbole/juno/blob/cosmos/v0.44.x/database/migrate/utils/utils.go#L4) and use their own branch...

Reply:
Oh you're right... ok let me update it